R/pkg/R/mllib.R

```diff
@@ -41,7 +41,8 @@ setClass("PipelineModel", representation(model = "jobj"))
 #' sqlContext <- sparkRSQL.init(sc)
 #' data(iris)
 #' df <- createDataFrame(sqlContext, iris)
-#' model <- glm(Sepal_Length ~ Sepal_Width, df)
+#' model <- glm(Sepal_Length ~ Sepal_Width, df, family=c("gaussian","binomial"))
```

Comment:
I see , I can change it to model <- glm(Sepal_Length ~ Sepal_Width, df, family="gaussian"), since it is an example. 
And also, it seems that there is a problem when I set: family="binomial", at least in my environment. An error is being thrown.